### PR TITLE
Create sensor.tof

### PR DESCRIPTION
**Pull request by josemotta**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/21230):** home-assistant/home-assistant#21230

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.tof.markdown [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+layout: page
+title: "Time of Flight sensor using VL53L1X"
+description: "Instructions on how to integrate a VL53L1X ToF sensor into Home Assistant."
+date: 2019-02-21 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: DIY
```
Comment (klaasnicolaas) [inline source/_components/sensor.tof.markdown]:
```suggestion
ha_category: 
  - DIY
  - Sensor
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/sensor.tof.markdown [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+date: 2019-02-21 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: 
+  - DIY
+  - Sensor
+ha_release: 0.87.1
```
Comment (klaasnicolaas) [inline source/_components/sensor.tof.markdown]:
```suggestion
ha_release: 0.89
```

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by josemotta** [comment]
Having some issues also at pull request at home assistant. Any help?

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant/pull/21230):** [home-assistant/home-assistant#21230](https://github.com/home-assistant/home-assistant/pull/21230)

**Comment by klaasnicolaas** [comment]
Why closing this PR?

**Comment by josemotta** [comment]
Sorry, wrong action. Please keep it! It´s my first time here I was trying to ask for help at code PR;

Thanks for your support

**Comment by josemotta** [comment]
reopening

**Comment by josemotta** [comment]
There is a broken link at top right:

Source: tof/sensor.py

Actually it should be `sensor/tof.py`, isn´t? How to fix this?

https://deploy-preview-8672--home-assistant-docs.netlify.com/components/sensor.tof/

**Comment by frenck** [comment]
This is known and has to do with "[The Great Migration](https://developers.home-assistant.io/blog/2019/02/19/the-great-migration.html)".

The documentation code currently only looks at the "new" situation.

From the documentation perspective, there is currently nothing you can do about that.

File: source/_components/sensor.tof.markdown [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+date: 2019-02-21 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: 
+  - DIY
+  - Sensor
+ha_release: 0.89
```
Comment (MartinHjelmare) [inline source/_components/sensor.tof.markdown]: "0.90"

Keep the quotes to keep the trailing zero.

File: source/_components/sensor.tof.markdown [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+ha_iot_class: "Local Push"
```
Comment (MartinHjelmare) [inline source/_components/sensor.tof.markdown]: Local Polling

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @josemotta! 👍

**Pull request merged**